cmd/anubis/main.go

```diff
@@ -217,6 +211,11 @@ func main() {
 			Action: config.RuleBenchmark,
 		}}
 	}
+	if *basePrefix != "" && !strings.HasPrefix(*basePrefix, "/") {
+		log.Fatalf("base-prefix must start with a slash") // could do a silent fix, but this is more transparent
```

Comment:
Good! Admin misconfigurations should always be loud and annoying!

```suggestion
		log.Fatalf("[misconfiguration] base-prefix must start with a slash, eg: /%s", *basePrefix)
```